packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -103,6 +131,14 @@ export class SerializedStateManager {
 	private refreshSnapshot: Promise<void> | undefined;
 	private readonly lastSavedOpSequenceNumber: number = 0;
 
+	/**
+	 * @param pendingLocalState - The pendingLocalState being rehydrated, if any (undefined when loading directly from storage)
+	 * @param subLogger - Container's logger to parent our logger under
```

Comment:
This comment is confusing to me - is the relationship that our logger is the parent of the parameter `subLogger`, or vice versa?

Reply:
Yeah I wondered if that wording was too sloppy.  I'll clarify.

Reply:
```suggestion
	 * @param subLogger - Container's logger to use as parent for our logger
```